packages/app/components/icon.component.tsx

```diff
@@ -23,3 +23,4 @@ export const SMSIcon = uiIcon('message-square-outline')
 export const SettingsIcon = uiIcon('settings-outline')
 export const BookOpenIcon = uiIcon('book-open-outline')
 export const GlobeIcon = uiIcon('globe-outline')
+export const ShareIcon = uiIcon('share-outline')
```

Comment:
The share icon is not commonly used on iOS. Consider change to browser-outline.
https://akveo.github.io/eva-icons/#/?searchKey=web&type=outline

Reply:
Isn't this more suitable in this case?
https://akveo.github.io/eva-icons/#/?searchKey=globe-outline&type=outline

Reply:
Alternatively this 
https://akveo.github.io/eva-icons/#/?searchKey=external&type=outline

Reply:
Yes! The last one external-link-outline I think is the best. The globe is used for localization on the login page.